README.org

```diff
@@ -882,15 +879,12 @@ The above code makes the backend available to select.  If you want it to be the
 #+begin_src emacs-lisp
 ;; OPTIONAL configuration
 (setq
- gptel-model   'grok-beta
+ gptel-model   'grok-3-latest
  gptel-backend
- (gptel-make-openai "xAI"           ;Any name you want
-   :host "api.x.ai"
+ (gptel-make-xai "xAI"           ;Any name you want
    :key "your-api-key"              ;can be a function that returns the key
-   :endpoint "/v1/chat/completions"
    :stream t
-   :models '(;; xAI now only offers `grok-beta` as of the time of this writing
-             grok-beta)))
+   :models 'grok-3-latest))
```

Comment:
This line is incorrect and also unnecessary now that we handle the models, it can be removed.